src/ol/geom/Geometry.js

```diff
@@ -92,7 +92,8 @@ class Geometry extends BaseObject {
    * @return {boolean} Contains (x, y).
    */
   containsXY(x, y) {
-    return false;
+    const coord = this.getClosestPoint([x, y]);
```

Comment:
We can do significantly less work than this function to determine intersection.  But this looks like a good start.